bin/tblextr.py

```diff
@@ -514,6 +561,9 @@ def fill_ops_db(table_name, db, indir):
     dform.post_process_data(db, 'OPS')
     dform.gen_ops_json_trace(db, 'OPS', GPU_BASE_PID, START_US, jsonfile)
 
+    sysinfo_file2 = re.sub(r'\.stats\.csv$', r'.sysinfo_stats2.txt', statfile)
+    params2 = json_metadata_gen(sysinfo_file2, 2)
+
```

Comment:
Need just one sysinfo file
And to append HIP info to the end
Should be:
metadata_gen(sysinfo_file, '/opt/rocm/bin/hipcc --version')

Reply:
I changed the code to have one sysinfo file: def metadata_gen(sysinfo_file, sysinfo_cmd)